lib/client/api.go

```diff
@@ -476,6 +512,11 @@ func (c *Config) LoadProfile(profileDir string, proxyName string) error {
 		log.Warnf("Error parsing user profile: %v", err)
 	}
 
+	c.DynamicForwardedPorts, err = ParseDynamicPortForwardSpec(cp.DynamicForwardedPorts)
+	if err != nil {
+		log.Warnf("Error parsing user profile: %v", err)
```

Comment:
missing dot. Also it's a warning, but log message says `error` what is not consistent.